tidb-cloud/tidb-cloud-security-faq.md

```diff
@@ -0,0 +1,91 @@
+---
+title: TiDB Cloud Security FAQs
+summary: Learn about the most frequently asked questions (FAQs) relating to TiDB Cloud security.
+---
+
+# TiDB Cloud Security FAQs
+
+This document lists the most frequently asked questions about TiDB Cloud security.
+
+## What security and privacy certifications has TiDB Cloud obtained?
+
+TiDB Cloud has obtained the following security and privacy protection certifications:
+
+- SOC 2 Type II
+- ISO/IEC 27001:2013
+- ISO/IEC 27701
+- General Data Protection Regulation (GDPR)
+
+For more information, see [PingCAP Trust & Compliance Center](https://en.pingcap.com/trust-compliance-center).
+
+## How does TiDB Cloud ensure the security of data transfer between client applications and TiDB clusters?
+
+TiDB Cloud provides the encryption ability of data transfer between client applications and TiDB clusters as follows:
+
+- TiDB Cloud supports [VPC Peering](/tidb-cloud/set-up-vpc-peering-connections.md), which ensures the security of the data transfer between applications and TiDB cluster.
+- For the connection to a TiDB Cloud cluster via a MySQL client, TiDB Cloud enables data transfer encryption by default.
```

Comment:
For the connection to a TiDB Cloud cluster via a MySQL client, TiDB Cloud cluster enables data transfer encryption by default.